lte/gateway/c/oai/tasks/sgw/sgw_handlers.c

```diff
@@ -1736,43 +1733,34 @@ int sgw_handle_s5_create_bearer_response(
   create_session_response_p->bearer_contexts_created.bearer_contexts[0]
     .cause.cause_value = cause;
   create_session_response_p->bearer_contexts_created.num_bearer_context += 1;
-  if (new_bearer_ctxt_info_p) {
-    create_session_response_p->teid =
-      new_bearer_ctxt_info_p->sgw_eps_bearer_context_information.mme_teid_S11;
-    create_session_response_p->trxn =
-      new_bearer_ctxt_info_p->sgw_eps_bearer_context_information.trxn;
-  }
+  create_session_response_p->teid =
+    new_bearer_ctxt_info_p->sgw_eps_bearer_context_information.mme_teid_S11;
+  create_session_response_p->trxn =
+    new_bearer_ctxt_info_p->sgw_eps_bearer_context_information.trxn;
+  message_p->ittiMsgHeader.imsi =
+    new_bearer_ctxt_info_p->sgw_eps_bearer_context_information.imsi64;
   OAILOG_DEBUG(
     LOG_SPGW_APP,
     "Sending S11 Create Session Response to MME, MME S11 teid = %u\n",
     create_session_response_p->teid);
 
-  message_p->ittiMsgHeader.imsi = imsi64;
```

Comment:
We should keep adding the IMSI to the ITTI msg header as it will be passed to MME for UE indexed logging and state by subscriber.

Reply:
Imsi64 is added to ITTI message header in line num, 1740. 
On reception of Create Session Request, imsi64 is stored into SGW's UE/bearer context. While sending Create Session Response, getting imsi64 from SGW's bearer context.
The stored imsi64 can be used for all other ITTI messages between MME and SPGW.

Reply:
Imsi64 is added to ITTI message header in line num, 1740. 
On reception of Create Session Request, imsi64 is stored into SGW's UE/bearer context. While sending Create Session Response, getting imsi64 from SGW's bearer context.
The stored imsi64 can be used for all other ITTI messages between MME and SPGW.

Reply:
Imsi64 is added to ITTI message header in line num, 1740. 
On reception of Create Session Request, imsi64 is stored into SGW's UE/bearer context. While sending Create Session Response, getting imsi64 from SGW's bearer context.
The stored imsi64 can be used for all other ITTI messages between MME and SPGW.

Reply:
Imsi64 is added to ITTI message header in line num, 1740. 
On reception of Create Session Request, imsi64 is stored into SGW's UE/bearer context. While sending Create Session Response, getting imsi64 from SGW's bearer context.
The stored imsi64 can be used for all other ITTI messages between MME and SPGW.

Reply:
SGW will contain and be able to retrieve IMSI, however, when sending Create Session Resp, the received ITTI message on MME will not contain IMSI. It could retrieve it from MME state context I agree, but this is mostly to have an easy method to retrieve IMSI before parsing state, so we can filter the state and not parse it for all UEs, so my suggestion would be to keep this assignment.

Reply:
We had two approaches for sending imsi64 in itti message header
1. On reception of create session request, get imsi and store as part of SGW UE/bearer context. For any further messages to be sent to MME shall get the imsi64 from stored SGW's uE/bearer context.
2. SGW's UE/bearer has imsi stored in Imsi_t format, while sending itti message to MME, SGW shall convert Imsi_t to imsi64 and send to MME.
Me and Shruti discussed on both approaches, we concluded on 1st approach, because second approach has conversion over head for each message to be sent to MME.

Reply:
We had two approaches for sending imsi64 in itti message header
1. On reception of create session request, get imsi and store as part of SGW UE/bearer context. For any further messages to be sent to MME shall get the imsi64 from stored SGW's uE/bearer context.
2. SGW's UE/bearer has imsi stored in Imsi_t format, while sending itti message to MME, SGW shall convert Imsi_t to imsi64 and send to MME.
Me and Shruti discussed on both approaches, we concluded on 1st approach, because second approach has conversion over head for each message to be sent to MME.

Reply:
I agree on approach 1, what I was referring to is that the IMSI still needs to be added to the ITTI message that's gonna be sent to MME task.